docs/assets/extra.css

```diff
@@ -66,25 +68,34 @@
 }
 
 .md-typeset table:not([class]) th {
-    background-color: rgba(0,184,212,.1);
+    background-color: rgba(0, 184, 212, 0.1);
 }
 
 .md-typeset table:not([class]) tbody tr:nth-child(even) td {
-    background-color: rgba(0,184,212,.05);
+    background-color: rgba(0, 184, 212, 0.05);
 }
 
 /* Display of get-a-demo link */
 
 .getademo {
-    display: inline-block;
-    padding: 0.4rem;
-    height: 1.8rem;
     background-color: #f39200;
-    color: #fff;
-    vertical-align: middle;
-    font-size: 0.9rem;
-    font-weight: 700;
+    border-radius: 999px;
+    box-shadow: #f39200 0 10px 20px -10px;
     box-sizing: border-box;
+    color: #ffffff;
+    cursor: pointer;
+    font-size: 0.7rem;
+    font-weight: 700;
+    line-height: 24px;
```

Comment:
Is there a reason why you switched to `px` values?

The whole page specified by relative size units (mostly `rem`) so we should not change this for only this on button. Also the form element for the search bar next to the button is set in `rem` height.

The correct matching of the `px` and `rem` height is only given for plain default configurations, for example if a user (or a system) configured different default font size then this will break.

So please switch back to `rem` unit for size values.